### PR TITLE
Added React to our project

### PR DESCRIPTION
**Pull request by VasilPaligorov**
Added React to our project

### CODE REVIEW

File: flaskProject1/static/js/Register/Register1.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    }
+
+    render() {
+        return (
+            <>
+
+            </>
+        )
+    }
+}
```
Comment (kiko1134) [inline flaskProject1/static/js/Register/Register1.js:16]: We must discuss this together!

File: flaskProject1/static/js/HomePage/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+                            <ul className="navbar-nav ml-auto flex-nowrap">
+                                <li className="nav-item">
+                                    <a href="/login" className="nav-link"> Sign in</a>
+                                </li>
+                                <li className="nav-item">
+                                    <a href="/register" className="nav-link"> Sign up</a>
+                                </li>
+                            </ul>
+                        </>
+                    }
```
Comment (kiko1134) [inline flaskProject1/static/js/HomePage/Navbar.js:65]: We have to find a proper solution for this problem

File: flaskProject1/static/js/Register/Register1.js (hunk 1 above)
Comment (VasilPaligorov) [reply flaskProject1/static/js/Register/Register1.js:16]: Sure boss

File: flaskProject1/static/js/HomePage/Navbar.js (hunk 2 above)
Comment (VasilPaligorov) [reply flaskProject1/static/js/HomePage/Navbar.js:65]: I'm thinking about it!

**Pull request merged**